libraries/RC_Channel/RC_Channel.h

```diff
@@ -163,6 +163,12 @@ class RC_Channel {
         GPS_DISABLE  =        65, // disable GPS for testing
         RELAY5 =              66, // Relay5 pin on/off
         RELAY6 =              67, // Relay6 pin on/off
+	STABILIZE =           68, // stabilize mode
```

Comment:
The coding style does not use tab codes.
White space is 4 characters.

Reply:
will do